WrathCombo/Combos/PvE/SAM/SAM_Helper.cs

```diff
@@ -1,22 +1,30 @@
-﻿using System.Linq;
+﻿#region
+
+using System.Linq;
 using Dalamud.Game.ClientState.JobGauge.Types;
 using ECommons.DalamudServices;
+using ECommons.GameFunctions;
 using FFXIVClientStructs.FFXIV.Client.Game;
 using WrathCombo.Combos.JobHelpers.Enums;
 using WrathCombo.Data;
 using static WrathCombo.CustomComboNS.Functions.CustomComboFunctions;
 
+#endregion
+
 namespace WrathCombo.Combos.PvE;
 
 internal partial class SAM
 {
     internal static SAMGauge gauge = GetJobGauge<SAMGauge>();
     internal static SAMOpenerLogic SAMOpener = new();
+    internal static bool TargetIsBoss = IsBoss(LocalPlayer.TargetObject);
+    internal static bool hasHostileTarget = HasTarget() && CurrentTarget.IsHostile();
 
```

Comment:
Might be best if these sort of things are taken out of class specific helpers and put into the custom combo stuff as overrides.

Reply:
Oh ye, I'll move em after dinner, then everyone can just use em. Smart cookie 😁